package.json

```diff
@@ -99,6 +82,6 @@
     ]
   },
   "engines": {
-    "node": ">= 14"
+    "node": ">= 18"
```

Comment:
This should be 16, not 18 #431

Reply:
@kfcampbell @wolfy1339 I'm confused over this change - _any_ change to `engines` is a breaking one. I know that it's since been reverted to v16, but that is only "less breaking" not "not breaking" - we should revert back to v14 and then do a new major if we want to change the min engine.

This has broken the workflow for [`unionfs`](https://github.com/streamich/unionfs/pull/758) since it is still testing against Node v14.

Reply:
I have opened #433 to revert this completely